packages/vite/src/generators/vitest/vitest-generator.ts

```diff
@@ -28,22 +28,13 @@ import {
 import initGenerator from '../init/init';
 import { VitestGeneratorSchema } from './schema';
 
-export function vitestGenerator(
-  tree: Tree,
-  schema: VitestGeneratorSchema,
-  hasPlugin = false
```

Comment:
This is apparently not used and as it is not officially part of the public API, I am not sure whether it should be considered a breaking change or not.

Reply:
Oh oups! I just noticed that Nuxt was using it.